Add Foundry references (#26)
scaffold-eth/se2-docs: docs/quick-start/environment.mdx

yarn chain
```

This command starts a local Ethereum network using Hardhat or Foundry, depending on which one you selected in the CLI. The network runs on your local machine and can be used for testing and development. You can customize the network configuration in:

- `packages/hardhat/hardhat.config.ts` if you have Hardhat as solidity framework.

> - `packages/foundry/foundry.toml` if you have Foundry as solidity framework.

Reviewer: Should we do it like below to keep the same formatting:

- Hardhat => `packages/hardhat/hardhat.config.ts`
- Foundry => `packages/foundry/foundry.toml`

yarn deploy
```

> This command deploys a test smart contract to the local network. The contract can be modified to suit your needs. Is located in:

Reviewer: I feel like this sentence is a bit unclear, how about:

"This command deploys a test smart contract to the local network. The contract can be modified to suit your needs and can be found in:"

- Hardhat => `packages/hardhat/contracts`
- Foundry => `packages/foundry/contracts`

> The `yarn deploy` command uses a deploy script to deploy the contract to the network. You can customize it. Is located in:

Reviewer: ```suggestion
The yarn deploy command uses a deploy script to deploy the contract to the network. You can customize the deployment script located in:
```

## What's Next:

- Edit your smart contract:
  - Hardhat => `YourContract.sol` in `packages/hardhat/contracts`
  - Foundry => `YourContract.sol` in `packages/foundry/contracts`
- Edit your deployment scripts:
  - Hardhat => `packages/hardhat/deploy`
  - Foundry => `packages/foundry/script`
- Edit your frontend in `packages/nextjs/pages`
- Edit the app config in `packages/nextjs/scaffold.config.ts`

> - Run smart contract test with `yarn hardhat:test`

Reviewer 2: Maybe point to `packages/hardhat/test` and use `yarn hardhat:test`  similarly for foundry `packages/foundry/test` => `yarn foundry:test`

Author: What looks better for you?

**Option 1.**
- Edit your smart contract test and run it:
  - Hardhat => `packages/hardhat/test` | `yarn hardhat:test`
  - Foundry => `packages/foundry/test` | `yarn foundry:test`

**Option 2.**
- Edit your smart contract test in:
  - Hardhat => `packages/hardhat/test`
  - Foundry => `packages/foundry/test`
- Run smart contract test with:
  - Hardhat => `yarn hardhat:test`
  - Foundry => `yarn foundry:test`

Author: For now I've implemented Option 2, but only to not do a push with both of them included 😊 If you like 1 better I'll update it, np!

Reviewer 2: lol confused both look good what if merge them ? : 
```
- Edit your smart contract test in:
  - Hardhat => `packages/hardhat/test` to run test use `yarn hardhat:test`
  - Foundry => `packages/foundry/test` to run test use `yarn foundry:test`
```

But even option 2 looks completely fine !! Please feel free to ignore this 🙌

Author: I've just pushed your final version 😊
